src/App/Zinc.DeveloperCenter.Client.Web/projects/zn-developercenter/src/app/screens/adr-list/adr-list.component.ts

```diff
@@ -1,16 +1,48 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { Router } from '@angular/router';
+import { Subject } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
+
+import { LoadingOverlayService } from '~/core/loading-module/services/loading-overlay/loading-overlay.service';
+import { Page } from '~/models/page.interface';
+import { ADRSummary, Repo } from '~/models/repo.interface';
+import { GitHubRepoService } from '~/shared/services/github-repo.service';
 
 @Component({
     selector: 'app-adr-list',
     templateUrl: './adr-list.component.html',
     styleUrls: ['./adr-list.component.scss']
 })
-export class AdrListComponent implements OnInit {
+export class AdrListComponent implements OnInit, OnDestroy {
+    public repos!: Page<Repo>;
+    public templateADRs!: ADRSummary[];
+    public fetchedRepos: boolean = false;
+
+    private destroyed$ = new Subject<void>();
 
-    public constructor() { }
+    public constructor(
+        private repoService: GitHubRepoService,
+        private loadingService: LoadingOverlayService
+    ) { }
 
     public ngOnInit(): void {
-        return;
+        this.getGSFSGitHubReposInit();
     }
 
+    public ngOnDestroy(): void {
+        this.destroyed$.next();
+        this.destroyed$.complete();
+    }
+
+    public getGSFSGitHubReposInit(): void
+    {
```

Comment:
Codacy has a fix for the issue: [Opening curly brace does not appear on the same line as controlling statement.](https://app.codacy.com/gh/GSFSGroup/Zinc.DeveloperCenter/pullRequest?prid=9672103)
```suggestion
        this.loadingService.show('Loading');
```